subsys/bluetooth/host/gatt.c

```diff
@@ -420,6 +420,13 @@ static void db_hash_gen(bool store)
 		return;
 	}
 
+	/**
+	 * Core 5.1, Vol 3, Part F, 3.3.1 says that multi-octet Characteristic
+	 * Values shall be LE unless otherwise defined in specification,
+	 * but bt_smp_aes_cmac uses BE data so we swap
```

Comment:
Id adjust reflect this text to include that although the spec doesn't states the endianess of the hash the PTS does expect it to be in little endianess while cmac always calculates the hash in big endianess.

Reply:
OK